wandb/sdk/launch/docker.py

```diff
@@ -231,7 +239,7 @@ def get_docker_command(
         _, _, port = _, _, port = api.settings("base_url").split(":")
         base_url = "http://host.docker.internal:{}".format(port)
     elif _is_wandb_dev_uri(api.settings("base_url")):
-        base_url = "http://host.docker.internal:9002"
+        base_url = "http://host.docker.internal:9003"
```

Comment:
out of curiosity what's the reason for this change?

Reply:
Recently the dev port changed from 9002 to 9003, this makes launch work with api.wandb.test